plugin/mlsdbonlinelocator.cpp

```diff
@@ -45,8 +46,22 @@ MlsdbOnlineLocator::MlsdbOnlineLocator(QObject *parent)
     , m_modemManager(new QOfonoExtModemManager(this))
     , m_simManager(0)
     , m_networkManager(new NetworkManager(this))
+    , m_fallbacksLacf(true)
+    , m_fallbacksIpf(true)
     , m_currentReply(0)
 {
+    QString MLSConfigFile = QStringLiteral("/etc/gps_xtra.ini");
+    QSettings settings(MLSConfigFile, QSettings::IniFormat);
+    if(settings.contains("MLS/FALLBACKS_LACF")) {
+        m_fallbacksLacf = settings.value("MLS/FALLBACKS_LACF", "").toBool();
+    }
+    if(settings.contains("MLS/FALLBACKS_IPF")) {
+        m_fallbacksIpf = settings.value("MLS/FALLBACKS_IPF", "").toBool();
+    }
+
+    qCDebug(lcGeoclueMlsdb) << "  MLS_FALLBACKS_LACF" << m_fallbacksLacf
```

Comment:
No need for extra space. qCDebug and qDebug will add spaces between words automatically. So, "  MLS_FALLBACKS_LACF" should be like "MLS_FALLBACKS_LACF"

Reply:
ok